misc/formatter/modules/formatter-core/src/test/java/org/ballerinalang/formatter/core/FormatterTest.java

```diff
@@ -59,8 +59,13 @@ public void test(String source, String sourcePath) throws IOException {
         String content = getSourceText(sourceFilePath);
         TextDocument textDocument = TextDocuments.from(content);
         SyntaxTree syntaxTree = SyntaxTree.from(textDocument);
-        SyntaxTree newSyntaxTree = Formatter.format(syntaxTree);
-        Assert.assertEquals(newSyntaxTree.toSourceCode(), getSourceText(assertFilePath));
+        try {
+            SyntaxTree newSyntaxTree = Formatter.format(syntaxTree);
+            Assert.assertEquals(newSyntaxTree.toSourceCode(), getSourceText(assertFilePath));
+        } catch (FormatterException e) {
+            // TODO: handle the test cases causing formatting exceptions
+//            Assert.fail(e.getMessage(), e);
```

Comment:
Are we going to add test cases for these particular negative cases or just ignore them? If ignoring let's remove the commented code

Reply:
These test cases should ideally be enabled since these are scenarios with no diagnostic issues (positive test cases).

But at the moment these are failing due to some parser issues which are yet to be fixed. Once they are fixed this can be enabled, and ideally, these test cases shouldn't fail.

Reply:
I think its better to fail the test here and disable any-failing tests, if there are issues from the parser. 
Otherwise, any newly failing test would go un-noticed .

Reply:
Done